api/async/publish_test.ts

```diff
@@ -238,10 +234,6 @@ Deno.test({
         },
         status: "success",
         message: "Published module.",
-        stats: {
-          total_files: 2,
-          total_size: 425,
-        },
```

Comment:
Does the frontend use these on `/status/<id>`?

Reply:
no